src/vscode-bicep/src/test/e2e/surveys.e2e.test.ts

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Survey } from "../../feedback/surveys";
+import { createActionContextMock } from "../mocks/actionContextMock";
+
+describe("surveys-e2etests", () => {
+  describe("getIsSurveyAvailable", () => {
+    it("active aka survey link", async () => {
+      const context = createActionContextMock();
+
+      const isAvailable = await Survey.getIsSurveyAvailable(
+        context,
+        // This test depends on this aka.ms link existing and being active (can point to https://github.com/Azure/bicep/)
+        "https://aka.ms/bicep/tests/surveytests/active"
+      );
+
+      expect(isAvailable).toBeTruthy();
+      expect(context.telemetry.properties.linkStatus).toBe("available");
```

Comment:
nit: can we change linkStatus -> surveyLinkStatus?

Reply:
c29ff54: cr feedback